CHANGELOG.md

```diff
@@ -62,11 +62,11 @@ NOTE: this version is a breaking change due to the removal of HPKP. Remove the H
 
 ## 5.0.2
 
-- Updates `Referrer-Policy` header to support multiple policy values
```

Comment:
We shouldn't retroactively changed the changelog here; it'll be difficult for folks to understand the timeline of changes. Instead, let's add a new release version that states that it lower cases all header keys.

Reply:
Addressed in https://github.com/github/secure_headers/pull/533/commits/8c5119668de47e00576908c879fa66a8fb41b9fa